### DISCUSSION TITLE
Is it possible to choose outgoing IP address?

### DISCUSSION THREAD

**Discussion by espressoelf** [emersion/maddy#385] [Q&A] [answered]
My server has multiple IP addresses and the one I want to use for Maddy isn't the default one. I did bind everything to the second IP address, so incoming connections work fine, but it uses the primary address for outgoing connections, which obviously causes problems (i.e. marked as spam).

Is it possible to tell Maddy what IP/interface to use for outgoing connections?

**Comment by foxcpp** [comment, marked as answer]
target.remote module has local_ip directive for this.
```
target.remote outbound_delivery {
  ...
  local_ip 1.2.3.4
}
```

**Reply by espressoelf** [reply]
Thanks, it works now. I should rtfm harder, sorry. Will mark as answer as soon as GitHub lets me (button disabled because of the tutorial which I can't close - "422 Unprocessable Entity").
